fbchat/client.py

```diff
@@ -469,14 +474,25 @@ def resetDefaultThread(self):
     END DEFAULT THREAD METHODS
     """ 
     def removeFriend(self, friend_id=None):
-        """param friend_id: The id of the friend that you want to remove"""
+        """
+        Removes a specifed friend from your friend list
+        :param friend_id: The id of the friend that you want to remove
+        :returns error when the removing was unsuccessful
+        :returns True when the removing was successful
+        """
         payload = {
             "friend_id": friend_id,
             "unref": "none",
             "confirm": "Confirm",
         }
         r = self._post(self.req_url.REMOVE_FRIEND,payload)
-        return r.ok
+        dict = parse_qs(urlparse(r.url).query)
```

Comment:
Be careful about overwriting `dict`, since its a builtin. Use a different, perhaps even a more descriptive name, like `query`